generator/sources/php53/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "kaltura/api-client-library",
```

Comment:
We want to submit it to https://packagist.org/, the php5.3 library already follows the latest php/composer standards. Do you want to add that name for backward compatibility or for forward compatibility?